### DISCUSSION TITLE
Mainnet minimum staking requirements

### DISCUSSION THREAD

**Discussion by raulillo82** [crypto-org-chain/chain-main#231] [Ideas] [answered]
Hi, I would like to contribute with some ideas in order to make the mainnet as open and functional as possible. Many of us are glad contributors to some of the previous testnets (thaler, or the one running, croese) and would like to keep our node working for the community and for a mainnet that promises to be awesome (we all believe this). However, there's the broad suspicion that there will be a high required_council_node_stake requirement to run the mainnet that will leave many people of the way. Moreover, it could leave out people who helped test the net since early stages.

Edit: this might also affect quality. If access is made easier for people who already showed that they're able to maintain a node is made easier, some bad quality nodes may be avoided. Not saying that people being able to afford large sums of tokens should be left out, only that they might not have proved to be valid to run a node so far.

The possibility of having a minimum stake of 500k CRO is unfortunately a no-go for many people who won't be able to afford such a high amount. Moreover, even people with sums that are close to that, have most of their stake tied to the crypto.com app (for the card and other benefits) or the exchange. Even if they/we decided to lose all that benefits for "only" having a node (which would be a bit unfair), the amount would be very possibly not enough.

All that having been said, I have some suggestions of possibilities in order not to leave out anyone. I'm sending some possiblities, some of which are not exclusive and could be even combined (see points 5 and 6, to be combined with some of the previous ones). Namely:

1- Remove the minimum required_council_node_stake requirement for everyone.
2- Remove the minimum required_council_node_stake requirement for users having participated in thaler/croese testnets.
3- Set a more reasonablerequired_council_node_stake requirement (let's say 50k?)
4- Apply 3 only to testnet validators.
5- Take into account stake of a user in either the exchange or the app (or both) in order to allow to run a node with a much lower starting required_council_node_stake requirement.
6- Last but no least, CdC could maybe implement the infrastructure for people to build pools for reaching whatever amount is required, in case it's as high as 500k cro: those would be written in a smartcontract (smartcontract that would only allow those funds to be tied into a node, details would need to be thought), ideally audited externally. that would be some work, but in exchange, CdC could charge some reasonable fee from the staking benefits of that node (so making a profit from this and still keeping the community happier). In case this has to be limited, priority would be given to, as usual, people having contributed in the testnet(s).
Maybe this goes nowhere, but i thought it was at least worth sharing... And of course, happy to read more additions from the community. Thanks in advance!

**Comment by diereysaa** [comment]
All these proposals are awesome @raulillo82, thanks for compiling them!

I would add some kind of extra reward for those who participated in the testnets, like an early entry time, so they (we) can start earning rewards before the big whales appear.

**Reply by jordicrypto** [reply]
Very good @raulillo82

**Comment by tomtau** [comment, marked as answer]
Hi @raulillo82 and @diereysaa , thank you for sharing your ideas. We really appreciate your feedback and any inputs the community may have on how to make the mainnet more rewarding and fair for all participants!
Do note that the current early testnet network parameters (not only staking parameters) are not indicative of what they will be on the mainnet.
If you mean `min-self-delegation`, that's up to each validator to decide what to signal, similarly to `commission-rate`.
As for post-testnet incentives based on testnet participation, that's something to be explored in later testnet phases, especially as part of our upcoming pre-mainnet chain competition! Stay tuned for more info on this!

**Reply by raulillo82** [reply]
No, I'm afraid I wasn't referring myself to min-self-delegation, but rather to required_council_node_stake
I will edit my post with the exact name. Is this something expected to be lowered from the current amount, which is half a million tokens?

**Reply by tomtau** [reply]
`required_council_node_stake` refers to https://github.com/crypto-com/thaler code, not https://github.com/crypto-com/chain-main/ ?

**Reply by raulillo82** [reply]
Leaving aside technical testnet/mainnet network parameters, I'd like to come back to the original discussion, which is whether users will need a big amount of CRO to deploy their node in the mainnet.
For example, this was 500000 (5e5) test cro's. Should that be the case, I did propose a good bunch of ideas to make it easier for validators having contributed to the testnet, which might be possible or not, but that could be considered for a discussion.

**Reply by antbelcastro** [reply]
Just echoing what @raulillo82 has mentioned here. Ideally it would be really good to get a ball park figure of what the minimum required stake would be. From here validators could plan accordingly and plan on how to acquire the minimum required CRO required to transfer their skills over from the testnet.

**Comment by jordicrypto** [comment]
We have been in the CRO crypto.com testnet project for months.  Here I don't think we are whales, it would be nice if they gave an answer to Raúl to organize themselves, we are all excited to enter the mainnet first.  Especially Raul and Diego have helped me a lot.  Let's see if they give us approximate data to organize ourselves

@raulillo82 @diereysaa
